language/ru/googleanalytics_acp.php

```diff
@@ -0,0 +1,46 @@
+<?php
```

Comment:
You mean this opening tag or the whole file? Looks like it was committed by mistake and should be added via another PR.